cmd/goal/network.go

```diff
@@ -80,10 +88,23 @@ var networkCreateCmd = &cobra.Command{
 		if err != nil {
 			panic(err)
 		}
-		networkTemplateFile, err := filepath.Abs(networkTemplateFile)
-		if err != nil {
-			panic(err)
+
+		var defaultReader io.Reader
+
+		if networkTemplateFile == "" {
+			if devModeOverride {
```

Comment:
if networkTemplateFile isn't provided in the arg, we can set a default path here?

Reply:
Would it be used for something?